### ISSUE TITLE
Notifications test is incorrect when built in isolation

### ISSUE THREAD

**Issue by jameshalsall** [Modernizr/the-old-modernizr.com#41]
When using the website to create a custom build of Modernizr, with only the `notification` check... it creates an invalid build.

Build URL: http://modernizr.com/download/#-cssclasses-prefixed-testprop-testallprops-domprefixes-notification

The offending line:

``` javascript
Modernizr.addTest("notification",!!Modernizr.prefixed("Notifications",window));
```

should be

``` javascript
Modernizr.addTest("notification",!!Modernizr.prefixed("Notification",window));
```

**Comment by patrickkettner** [comment]
This isn't a problem with the builder, but rather the test is older. It was fixed in [#1015](https://github.com/Modernizr/modernizr.com/issues/1015). This should be backported in the next release.

Thank you for opening the issue!

**Comment by patrickkettner** [comment]
This was released in v2.8.0
